### PR TITLE
Release 34 Update

### PR DESCRIPTION
**Pull request by edobry**
Just some updates made during the release process.

> ProTip: [markdown-preview-enhanced](https://atom.io/packages/markdown-preview-enhanced) is your friend

### CODE REVIEW

File: process/Development.md [hunk 1]
```diff
@@ -21,5 +21,6 @@ You are responsible for shepherding your work through the development process. I
    - make the changes
    - notify the reviewer by tagging them in a comment
 - **Once you have two approvals and one test:**
+- Label your PR **ready**
```
Comment (fcamilleri22) [inline process/Development.md:24]: in Github -- may be confused with the ready column in JIRA

File: process/Release.md [hunk 2]
```diff
@@ -1,20 +1,29 @@
 **Scenario:** You want to release a new version
 
-- Add issues to release on JIRA
+- Notify team of release start
+- Set release start date to today
```
Comment (fcamilleri22) [inline process/Release.md:4]: In "[JIRA Project > Releases]"?

That one's up to you, I feel like we should be -slightly- specific about directions, but not too too specific, considering that the technologies used in our process are very much still in flux.

**Review by fcamilleri22 — CHANGES_REQUESTED** [review]
Small comments, and also a few more comments I'd like in this PR that I don't think have been brought up elsewhere. I assure you, nothing massive or overly deep.

**Comment by fcamilleri22** [comment]
In format/issues.md:
There's some confusion in the Location notation section of issues. You first use the "-" for location-thing, but then use ">" in the example. ">" should be mentioned around there to mean "hierarchy"

**Comment by fcamilleri22** [comment]
In format/issues.md:
I feel like the "Bugs" section is very easily missable. Maybe extract it into its own file, and mention in the file that "for other unmentioned fields, follow issues.md" Up to you on that one.

File: process/Development.md (hunk 1 above)
Comment (edobry) [reply process/Development.md:24]: i thought this was unambiguous, as there is no such thing as a JIRA PR

File: process/Release.md (hunk 2 above)
Comment (edobry) [reply process/Release.md:4]: there are several places to do this, and they are all equivalent. is there a benefit to specifying a specific one?

**Comment by fcamilleri22** [comment]
Less so because it's actually needed, moreso for lulz.
In process/deployment.md, can you, alongside touch butt, mention "man touch?"

That would make me suuuper happy.

**Comment by fcamilleri22** [comment]
process/whatDo?.md

Make it clearer that that's supposed to be a prioritized list, blockers first, then supporting existing work, yadda yadda

**Comment by edobry** [comment]
@fcamilleri22 no :|

File: process/Development.md (hunk 1 above)
Comment (fcamilleri22) [reply process/Development.md:24]: I feel like PR can be conflated to be the same as issue. It's inaccurate thought, sure, but if we can prevent it, lets.

Comment (fcamilleri22) [reply process/Development.md:24]: -resolved-

File: process/Release.md (hunk 2 above)
Comment (fcamilleri22) [reply process/Release.md:4]: That's just moreso thinking from my experience -- I'm usually only handling the JIRA thingy. It seems that there are advantages to making it intentionally broad -- i'll make it your call.

**Review by fcamilleri22 — APPROVED** [review]

**Pull request merged**